modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go

```diff
@@ -102,7 +102,7 @@ func (o *OrgFormModal) SetProps() {
 				"rules": []interface{}{
 					map[string]interface{}{
 						"msg":     "由小写字母、数字，-组成(不允许纯数字)",
-						"pattern": "/^(?![0-9]+$)^[^-](.*[^-])?$[a-z0-9-]*$/",
+						"pattern": "/^(?:[a-z]+|[0-9]+[a-z]+|[0-9]+[-]+[a-z0-9])+(?:(?:(?:[-]*)[a-z0-9]+)+)?$/",
```

Comment:
use const and add unit test for this.

Reply:
this regexp is for js, it's hard to unit test.

and we are planing disable frontend check, all check will in backend.